apps/meteor/packages/meteor-run-as-user/lib/collection.overwrites.js

```diff
@@ -1,3 +1,38 @@
+// TODO: Remove this after meteor fixes the issue
+// This override fixes a recent issue introduced after the update to Meteor 3.0
```

Comment:
Was this already reported to them? If so, it would be good to add the issue here, and also the piece of code with the bug, if you were able to track it in their code...

Reply:
https://github.com/meteor/meteor/pull/13440